images/win/scripts/SoftwareReport/SoftwareReport.Common.psm1

```diff
@@ -214,7 +220,7 @@ function Get-DotnetRuntimes {
     $runtimesRawList = dotnet --list-runtimes
     $runtimesRawList | Group-Object {$_.Split()[0]} | ForEach-Object {
         $runtimeName = $_.Name
-        $runtimeVersions = ($_.Group | Foreach-Object {$_.split()[1]}) -join ', '
+        $runtimeVersions = ($_.Group | Foreach-Object {$_.split()[1]}) -join ' '
```

Comment:
I think we don't need to make join at all. We need $runtimeVersions to be the array of versions instead of the single string.

Reply:
corrected, my fault again